dataset/src/utils.ts

```diff
@@ -80,3 +89,40 @@ export class ExhaustiveError extends Error {
     super(message ?? `Unexpected value: ${JSON.stringify(value, null, 2)}`);
   }
 }
+
+/** 単語をアルファベット読み（エービーシー）に変換する */
+export const wordToAlphabetPronunciation = (word: string): string => {
+  return word
+    .split("")
+    .map((c) => alphabetMap[c.toLowerCase() as keyof typeof alphabetMap])
+    .join("");
+};
+
+export const alphabetMap = {
```

Comment:
```suggestion
const alphabetMap = {
```

YAGNI的にまあ「exportしなくて良いものはしない方が良い」が正解かなーと